src/api/test/unit/role_test.rb

```diff
@@ -1,15 +0,0 @@
-require_relative '../test_helper'
-
-class RoleTest < ActiveSupport::TestCase
-  def test_something
-    norole = Role.create(title: 'norole')
-    norole.title = 'thisrole'
-    norole.save
-    norole.destroy
-  end
-
-  def test_role
```

Comment:
Should we test this with `validates_uniqueness_of` from `shoulda-matchers`? 

I don't really see benefits in doing so, but I'm interesting in knowing what you think.

Reply:
Testing that rails features work makes little sense. The rails test suite tests that...